ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2913,18 +2876,32 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-workers-rhel8-fips
-- as: gcp-ipi-workload-identity-fips-p1-f28-destructive
-  cron: 2 3 3 * *
+- as: gcp-ipi-workload-identity-rt-fips-p1-f28-destructive
```

Comment:
suggest to change `p1` to `p3`, and swap the order of `gcp-ipi-workload-identity-rt-fips-p1-f28-destructive` and `gcp-ipi-workload-identity-rt-fips-p2-f14`